hg-fast-export.py

```diff
@@ -484,6 +485,7 @@ def verify_heads(ui,repo,cache,force,branchesmap):
 
   # get list of hg's branches to verify, don't take all git has
   for _,_,b in l:
+    b=b.decode(branch_encoding).encode('utf8')
```

Comment:
It would be take default global encoding, this is full analog, as fn_encoding.

```python
  branch_encoding=encoding
  if options.branch_encoding!=None:
    branch_encoding=options.branch_encoding
```

Reply:
Se above